powersimdata/scenario/create.py

```diff
@@ -176,6 +173,9 @@ def set_grid(self, grid_model="usa_tamu", interconnect="USA"):
         self._scenario_info["grid_model"] = self.builder.grid_model
         self._scenario_info["interconnect"] = self.builder.interconnect
 
+    def _leave(self):
```

Comment:
missing docstring. You can use `"""Cleans when leaving state."""` as used in the `Analyze` classs